From 084a57b3c055a48be468e2b2738b7b8a1e4ff06b Mon Sep 17 00:00:00 2001
From: Klaus Zimmermann <klaus.zimmermann@quansight.com>
Date: Tue, 27 Feb 2024 10:53:05 +0100
Subject: [PATCH] Remove-march-flags

---
 GKlibSystem.cmake | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/GKlibSystem.cmake b/GKlibSystem.cmake
index 31a1cf1..88fca25 100644
--- a/GKlibSystem.cmake
+++ b/GKlibSystem.cmake
@@ -31,16 +31,6 @@ endif(CYGWIN)
 if(CMAKE_COMPILER_IS_GNUCC)
 # GCC opts.
   set(GKlib_COPTIONS "${GKlib_COPTIONS} -std=c99 -fno-strict-aliasing")
-if(VALGRIND)
-  set(GKlib_COPTIONS "${GK_COPTIONS} -march=x86-64 -mtune=generic")
-else()
-# -march=native is not a valid flag on PPC:
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "power|ppc|powerpc|ppc64|powerpc64" OR (APPLE AND CMAKE_OSX_ARCHITECTURES MATCHES "ppc|ppc64"))
-  set(GKlib_COPTIONS "${GKlib_COPTIONS} -mtune=native")
-else()
-  set(GKlib_COPTIONS "${GKlib_COPTIONS} -march=native")
-endif()
-endif(VALGRIND)
   if(NOT MINGW)
       set(GKlib_COPTIONS "${GKlib_COPTIONS} -fPIC")
   endif(NOT MINGW)
-- 
2.40.1

